.circleci/config.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       - run: RENOVATE_CONFIG_FILE=web.json npx -p renovate renovate-config-validator
       - run: RENOVATE_CONFIG_FILE=internalSmgPackageUpdate.json npx -p renovate renovate-config-validator
       - run: RENOVATE_CONFIG_FILE=groupDevDependencies.json npx -p renovate renovate-config-validator
-      - run: RENOVATE_CONFIG_FILE=autoMergeNonMajorNode.json npx -p renovate renovate-config-validator
+      - run: RENOVATE_CONFIG_FILE=autoMergeLtsNodeVersions.json npx -p renovate renovate-config-validator
       - run: RENOVATE_CONFIG_FILE=autoMergeNonMajorDocker.json npx -p renovate renovate-config-validator
```

Comment:
seems odd that we need to run the command multiple times with different configs. unrelated to this PR, but maybe we can keep our eyes open to see if there is any way to roll with one execution with multiple configs. or a shared config that references more detailed configs

this kinda is a maintenance issue as we'll have to keep this CI step in sync across multiple projects. simpler is always less prone to errors...

Reply:
this step is not in other projects. The config is shared automatically

Reply:
mea culpa. I mixed up the context of this change. it's in the renovate config, not a project